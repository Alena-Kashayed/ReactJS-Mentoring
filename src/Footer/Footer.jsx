import React from 'react';
import styles from './Footer.scss';

function Footer() {
  return (
    <div className={styles.footer}>
      <span className={styles.logo}>Netflixtrullete</span>
    </div>
  );
}

export default Footer;
