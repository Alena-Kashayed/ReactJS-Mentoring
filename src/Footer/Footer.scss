.footer {
  height: 80px;
  background-color: #414141;
  display: flex;
  align-items: center;
  padding: 0 50px;
  color: #F55263;
  font-weight: 600;

  .logo {
    max-width: 1200px;
  }
}
