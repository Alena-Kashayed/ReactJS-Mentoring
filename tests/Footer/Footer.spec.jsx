import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import Footer from '../../src/Footer/Footer';

describe('Footer', () => {
  it('is rendered correctly', () => {
    const wrapper = shallow(<Footer />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
