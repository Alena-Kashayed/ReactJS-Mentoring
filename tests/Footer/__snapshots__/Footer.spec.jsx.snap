// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer is rendered correctly 1`] = `
<div
  className="footer"
>
  <div
    className="logo"
  >
    Netflixtrullete
  </div>
</div>
`;
